### DISCUSSION TITLE
how to change the tigger widget?

### DISCUSSION THREAD

**Discussion by JasonWithJson** [davigmacode/flutter_choice#2] [Q&A]
how to change the tigger widget

**Comment by davigmacode** [comment]
To change the trigger widget, you can use the [`anchorBuilder`](https://pub.dev/documentation/choice/latest/choice_prompt/PromptedChoice/anchorBuilder.html) property. This property takes a builder function that creates the widget that will be used as the trigger.

```dart
PromptedChoice(
  ...,
  anchorBuilder: (state, openPrompt) {
    return YourTriggerWidget(
      onTap: openPrompt,
    );
  },
)
```
Also, you can use [the customizable default trigger builder](https://pub.dev/documentation/choice/latest/choice_anchor/ChoiceAnchor/create.html)
```dart
PromptedChoice(
  ...,
  anchorBuilder: ChoiceAnchor.create(valueTruncate: 1),
)
```
